File: SecondYear/gamandeepsingh_GamandeepSingh_2226csit1042_2/Week_2/Armstrong.java
import java.util.*;
public class Main {
	
	public static void main(String[] args) {
		// Write your code here
		Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
		int c=(int)(Math.log10(n) + 1);
		int m=n;
		int sum=0;
        while(m!=0){
			sum=sum+(int)Math.pow((m%10),c);
			m/=10;
		} 
		if(sum==n)
		  System.out.println("true");
		else
          System.out.println("false"); 
	}
}

File: SecondYear/seeeyaa_PariGupta_2226CSE(AI)1096_2/Week_2/Armstrong.cpp
bool checkArmstrong(int n)
{
	//Write your code here
	int n1=n,n2=n,count=0,sum=0,exp,rem;
	if(n<0) return 0 ;
	while(n1!=0)
	{
		n1/=10;
		count++;
	}
	while(n2!=0)
	{
		rem=n2%10;
		exp=pow(rem,count);
		sum+=exp;
		n2/=10;
	}
	if (sum==n) return 1;
	else return 0;


}
